L1Trigger/L1TCaloLayer1/test/testUCTLayer1.cpp

```diff
@@ -57,7 +57,7 @@ void print(UCTLayer1& uct) {
         if (regions[rgn]->et() > 0) {
           int hitEta = regions[rgn]->hitCaloEta();
           int hitPhi = regions[rgn]->hitCaloPhi();
-          vector<UCTTower*> towers = regions[rgn]->getTowers();
+          std::vector<std::shared_ptr<UCTTower>> towers = regions[rgn]->getTowers();
```

Comment:
```suggestion
          auto const& towers = regions[rgn]->getTowers();
```
will be easier to maintain and will be more efficient as it avoids a copy.

Reply:
Applied in most recent commit